### PR TITLE
Update output error message (more clear output)

### PR DESCRIPTION
**Pull request by tomassebestik**
This PR updates output message to make it more clear and easy to read.

**BEFORE:**
 

**AFTER:**

 

With optional args:
 

### Other changes
- ci: update project settings configuration (pyproject.toml)

## Related
- https://github.com/espressif/conventional-precommit-linter/issues/8

### CODE REVIEW

**Comment by tomassebestik** [comment]
@hfudev PTAL

File: conventional_precommit_linter/helpers.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+def _color_bold_green(text: str) -> str:
```
Comment (hfudev) [inline conventional_precommit_linter/helpers.py:1]: if we consider multi-os support, here you may use `colorama` or `rich`.

**Review by hfudev — APPROVED** [review]
@tomassebestik LGTM! The output is exactly what I need!

File: conventional_precommit_linter/helpers.py (hunk 1 above)
Comment (tomassebestik) [reply conventional_precommit_linter/helpers.py]: Thanks! Implemented with `colorama`.

**Pull request merged**